go.mod

```diff
@@ -108,10 +108,10 @@ require (
 	github.com/valyala/fasttemplate v1.1.0 // indirect
 	github.com/vmihailenco/msgpack/v5 v5.0.0-beta.1
 	go.opencensus.io v0.22.3
-	go.packetbroker.org/api/iam v1.2.2-tts
-	go.packetbroker.org/api/iam/v2 v2.1.2-tts
-	go.packetbroker.org/api/routing v1.2.2-tts
-	go.packetbroker.org/api/v3 v3.4.2-tts
+	go.packetbroker.org/api/iam v1.4.2-tts
```

Comment:
Needs `go mod tidy` in `tools`.

Reply:
It needed a `go mod download` in root and in `tools` ....